spec/features/ephemera_folder_spec.rb

```diff
@@ -67,7 +68,7 @@
 
   context 'within an existing ephemera box' do
     scenario 'users can save a new folder' do
-      visit parent_new_ephemera_box_path(parent_id: ephemera_box.id)
+      visit parent_new_ephemera_box_path(parent_id: ephemera_folder.id)
```

Comment:
Shouldn't the parent be an ephemera box here?  The context says "within an existing ephemera box".